sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -3108,7 +3108,8 @@ pi_result piEnqueueMemUnmap(pi_queue Queue, pi_mem MemObj, void *MappedPtr,
   // piEnqueueMemBufferMap, but can only do so after the above copy
   // is completed. Instead of waiting for It here (blocking), we shall
   // do so in piEventRelease called for the pi_event tracking the unmap.
-  (*Event)->CommandData = MemObj->MapHostPtr ? nullptr : MappedPtr;
+  if (Event)
+    (*Event)->CommandData = MemObj->MapHostPtr ? nullptr : MappedPtr;
```

Comment:
I think it would be at least more robust to init CommandData to null, if Event is null